### PR TITLE
Adding more work on salt encryption

### PR DESCRIPTION
**Pull request by cody-constine-ttd**

### CODE REVIEW

File: src/main/java/com/uid2/admin/store/writer/SaltStoreWriter.java [hunk 1]
```diff
@@ -130,7 +141,13 @@ private void uploadSaltsSnapshot(RotatingSaltProvider.SaltSnapshot snapshot, Str
             }
         }
 
-        cloudStorage.upload(newSaltsFile.toString(), location, this.currentTags);
+        //cloudStorage.upload(newSaltsFile.toString(), location, this.currentTags);
```
Comment (thomasm-ttd) [inline src/main/java/com/uid2/admin/store/writer/SaltStoreWriter.java:144]: remove the comment

**Review by thomasm-ttd — COMMENTED** [review]
Are you still adding tests for these?

File: src/main/java/com/uid2/admin/store/factory/SaltStoreFactory.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    @Override
+    public RotatingCloudEncryptionKeyProvider getCloudEncryptionProvider() {
+        return cloudEncryptionKeyProvider;
+    }
+
+    @Override
+    public StoreReader<Collection<RotatingSaltProvider.SaltSnapshot>> getReader(Integer siteId) {
+        return null;
```
[the remark this answers is not in the document]
Comment (cody-constine-ttd) [reply src/main/java/com/uid2/admin/store/factory/SaltStoreFactory.java:58]: Throwing would change the interface too. But I think splitting it up would also be annoying. 
I am thinking when we continue with Admin rework, we migrate salts to follow the pattern of our other files and we can fill these in then.

[the remark this answers is not in the document]
Comment (thomasm-ttd) [reply src/main/java/com/uid2/admin/store/factory/SaltStoreFactory.java:58]: Ok, lets leave it as, but add logging (at WARN level) to indicate the method is being called.

**Review by thomasm-ttd — APPROVED** [review]
Small comment about adding some logging

**Pull request merged**